chain/sync_test.go

```diff
@@ -1037,81 +1035,6 @@ func TestSyncCheckpointEarlierThanHead(t *testing.T) {
 	require.True(tu.t, p1Head.Equals(b.TipSet()))
 }
 
-func TestDrandNull(t *testing.T) {
```

Comment:
This test was pretty bad...better test in rand_test now that also covers this case